uplink/src/base/mod.rs

```diff
@@ -43,6 +43,13 @@ pub struct Ota {
     pub path: String,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct Stats {
+    pub enabled: bool,
+    pub names: Vec<String>,
```

Comment:
`names` is confusing. If this is process names, can you rename this to `process_names`?